Tasks/VsTest/vstest.ts

```diff
@@ -332,7 +349,7 @@ function runVStest(testResultsDirectory: string, settingsFile: string, vsVersion
     }
     else {
         tl.debug("Non TIA mode of test execution");
-        executeVstest(testResultsDirectory, settingsFile, vsVersion, getVstestArguments(settingsFile))
+        executeVstest(testResultsDirectory, settingsFile, vsVersion, getVstestArguments(settingsFile, false))
```

Comment:
Nope. Not as of now


---
In reply to: [74879634](https://github.com/Microsoft/vsts-tasks/pull/2264#discussion_r74879634) [](ancestors = 74879634)